### ISSUE TITLE
test_historical_workflow.py test failed

### ISSUE THREAD

**Issue by YeChen-IDM** [InstituteforDiseaseModeling/emod-genepi-archetypes#13]
```
test_historical_workflow.py:29: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../run_sims/create_sim_sweeps.py:129: in create_and_run_sim_sweep
    experiment.run(wait_until_done=True, platform=platform)
/usr/lib/python3.9/site-packages/idmtools/entities/experiment.py:560: in run
    p.run_items(self, **run_opts)
/usr/lib/python3.9/site-packages/idmtools/entities/iplatform.py:509: in run_items
    getattr(self, interface).run_item(item, **kwargs)
/usr/lib/python3.9/site-packages/idmtools/entities/iplatform_ops/iplatform_experiment_operations.py:268: in run_item
    self.pre_run_item(experiment, **kwargs)
/usr/lib/python3.9/site-packages/idmtools/entities/iplatform_ops/iplatform_experiment_operations.py:228: in pre_run_item
    experiment.simulations = self.platform._create_items_of_type(experiment.simulations, ItemType.SIMULATION,
/usr/lib/python3.9/site-packages/idmtools/entities/iplatform.py:471: in _create_items_of_type
    ni = getattr(self, interface).batch_create(items, **kwargs)
/usr/lib/python3.9/site-packages/idmtools_platform_comps/comps_operations/simulation_operations.py:275: in batch_create
    results = batch_create_items(
/usr/lib/python3.9/site-packages/idmtools/entities/iplatform_ops/utils.py:146: in batch_create_items
    for chunk in chunked(i, _batch_size):
/usr/lib/python3.9/site-packages/more_itertools/recipes.py:86: in take
    return list(islice(iterable, n))
/usr/lib/python3.9/site-packages/idmtools/entities/templated_simulation.py:46: in simulation_generator
    new_tags = func(simulation=simulation)
../run_sims/sweeps/historical_archetype_sweeps.py:45: in master_sweep_over_historical_scenarios
    simulation.task.create_campaign_from_callback(campaign_builder)
/usr/lib/python3.9/site-packages/emodpy/emod_task.py:124: in create_campaign_from_callback
    campaign = builder()
../run_sims/sweeps/historical_archetype_sweeps.py:33: in _campaign_builder
    campaign = build_full_maka_campaign()
../run_sims/archetypes/maka/maka_campaign.py:412: in build_full_maka_campaign
    add_healthseeking(campaign, sim_start_year=sim_start_year, hs_rate_factor=hs_rate_factor)
../run_sims/archetypes/maka/maka_campaign.py:306: in add_healthseeking
    add_treatment_seeking(campaign=campaign,
/usr/lib/python3.9/site-packages/emodpy_malaria/interventions/treatment_seeking.py:140: in add_treatment_seeking
    camp_events = _get_events(campaign=campaign, start_day=start_day, targets=targets, drug=drug, node_ids=node_ids,
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

campaign = <module 'emod_api.campaign' from '/usr/lib/python3.9/site-packages/emod_api/campaign.py'>
start_day = 17532
targets = [{'agemax': 5, 'agemin': 0, 'coverage': 0.4, 'rate': 0.3, ...}, {'agemax': 100, 'agemin': 5, 'coverage': 0.2, 'rate': ...': 5, 'agemin': 0, 'coverage': 0.9, 'rate': 0.5, ...}, {'agemax': 100, 'agemin': 5, 'coverage': 0.8, 'rate': 0.5, ...}]
drug = ['Artemether', 'Lumefantrine'], node_ids = None
ind_property_restrictions = None, drug_ineligibility_duration = 0, duration = -1
broadcast_event_name = 'Received_Treatment'

    def _get_events(
            campaign,
            start_day: int = 1,
            targets: list = None,
            drug: list = None,
            node_ids: list = None,
            ind_property_restrictions: list = None,
            drug_ineligibility_duration: float = 0,
            duration: int = -1,
            broadcast_event_name: str = 'Received_Treatment'):
        if not drug:
            drug = ['Artemether', 'Lumefantrine']
    
        if not targets:
            raise ValueError("Please define targets for treatment seeking. It is a list of dictionaries:\n "
                             "ex: [{\"trigger\":\"NewClinicalCase\", \"coverage\":0.8, \"agemin\":15, \"agemax\":70, \"rate\":0.3}]\n")
        for target in targets:
            if "trigger" not in target:
                raise ValueError("Please define \"trigger\" for each target dictionary. \n"
                                 "ex: [{\"trigger\":\"NewClinicalCase\", \"coverage\":0.7, \"agemax\":3 }]")
            if "seek" in target:
>               raise ValueError("Notice: \"seek\" parameter has been removed. Please remove it from your \"targets\""
                                 " dictionary."
                                 " Please modify the \"coverage\" parameter "
                                 "directly to attain a different coverage for the intervention. Previously, "
                                 "\"Demographic_Coverage\" was \"coverage\"x\"seek\". It is now just \"coverage\".\n")
E               ValueError: Notice: "seek" parameter has been removed. Please remove it from your "targets" dictionary. Please modify the "coverage" parameter directly to attain a different coverage for the intervention. Previously, "Demographic_Coverage" was "coverage"x"seek". It is now just "coverage".
```

**Comment by YeChen-IDM** [comment]
https://jenkins.apps.idmod.org/view/MPG/job/emod-genepi-archetypes_pipeline/40/console

**Comment by YeChen-IDM** [comment]
fix in PR #12
